### PR TITLE
Simple ota application

### PR DESCRIPTION
**Pull request by tdz**

### CODE REVIEW

**Comment by tdz** [comment]
@fabricedesre Here's a PR with the simple-ota example. It'll fetch an update over HTTP and apply it to the device. It's based on the code of the offline updater. The PR is not for merging yet, but I'll update together with the offline-updater PR. Just want you to have the patch available.

File: examples/simple-ota.rs [hunk 1]
```diff
@@ -0,0 +1,367 @@
...
+    }
+    println!("Device started as STATION");
+
+    try!(wlan_connect());
+
+    println!("Connection established w/ AP and IP is acquired");
+    Ok(())
+}
+
+fn http_demo() -> Result<(), Error> {
```
Comment (fabricedesre) [inline examples/simple-ota.rs:319]: nit: rename to simple_ota_demo

**Review by fabricedesre — APPROVED** [review]
lgtm!

**Comment by tdz** [comment]
This patch set has seen several changes and is now part of PR #60.

**Pull request closed without merging**